internal/pkg/manifest/workload.go

```diff
@@ -272,6 +273,18 @@ func toStringSlice(s *stringSliceOrString) ([]string, error) {
 	return out, nil
 }
 
+func toString(s *stringSliceOrString) (string, error) {
```

Comment:
Maybe a comment showing an example since we specifically use " " as the separator

Reply:
I removed this altogether, leaving `start_command` as `string` only (not `stringSliceOrString`). There is currently no good way to convert a slice to a string using shlex rules, and if we shouldn't do the conversion if we can't do it properly.